reaktive/src/jvmJsCommonMain/kotlin/com/badoo/reaktive/disposable/DisposableWrapper.kt

```diff
@@ -31,10 +31,33 @@ actual open class DisposableWrapper actual constructor() : Disposable {
      * Also disposes any replaced [Disposable].
      */
     actual fun set(disposable: Disposable?) {
+        replace(disposable)
+            ?.dispose()
+    }
+
+    /**
+     * Atomically either replaces any existing [Disposable]
+     * with the specified one or disposes it if wrapper is already disposed.
+     * Does not dispose any replaced [Disposable].
+     *
+     * @param disposable a new [Disposable], will be disposed if wrapper is already dispose
+     * @return replaced [Disposable] if any
+     */
```

Comment:
Remove doc? Expect already has it.

Reply:
Unfortunately docs are not exported for actuals and ate missing then